packages/spl/src/payment-router/PaymentRouterProgram.ts

```diff
@@ -0,0 +1,61 @@
+import { BN, Program, Provider } from '@coral-xyz/anchor'
+import { TOKEN_PROGRAM_ID } from '@solana/spl-token'
+import { PublicKey, SystemProgram } from '@solana/web3.js'
+
+import { PAYMENT_ROUTER_PROGRAM_ID } from './constants'
+import { IDL } from './paymentRouter'
+import { CreateRouteInstructionParams } from './types'
+
+export class PaymentRouterProgram {
+  public static readonly programId = new PublicKey(
```

Comment:
this should be configured by environment right? staging being `sp38CXGL9FoWPp9Avo4fevewEX4UqNkTSTFUPpQFRry`

Reply:
yes, it will be set in the args to each instruction creation - this is pattern matched to other SPL programs that default to their main-net addresses and provide handy constants to their main-net addresses